ci-operator/step-registry/openshift/e2e/cert-rotation/suspend/openshift-e2e-cert-rotation-suspend-commands.sh

```diff
@@ -52,12 +52,12 @@ copy-file-from-first-master "${KUBECONFIG_REMOTE}" "${KUBECONFIG_REMOTE}"
 
 # Set date for host
 sudo timedatectl status
-sudo timedatectl set-time ${SKEW}
+sudo timedatectl set-time +${SKEW}
 sudo timedatectl status
 
 # Skew clock on every node
 # TODO: Suspend, resume and make it resync time from host instead?
-run-on-all-nodes "timedatectl set-time ${SKEW} && timedatectl status"
+run-on-all-nodes "timedatectl set-time +${SKEW} && timedatectl status"
```

Comment:
We'd better keep the same Time Zone between host and nodes.
```suggestion
TZ=$(date +"%Z %z" | cut -d' ' -f1)
run-on-all-nodes "timedatectl set-timezone ${TZ} && timedatectl set-time +${SKEW} && timedatectl status"
```

Reply:
The nodes here are VMs running on a single host machine, so their timezones will match.

Just to be sure I'll add "sync timezones" once in prepare-for-shutdown step